extensions/amp-story/1.0/amp-story.js

```diff
@@ -1455,6 +1460,39 @@ export class AmpStory extends AMP.BaseElement {
     }
   }
 
+  /**
+   * @private
+   */
+  openMask_() {
+    const actions = Services.actionServiceForDoc(this.getAmpDoc());
+    if (!this.maskElement_) {
+      const mask = this.win.document.createElement('div');
+      mask.classList.add('i-amphtml-amp-story-sidebar-mask');
```

Comment:
Nit: `.i-amphtml-story-opacity-mask` (we don't add the `amp-` prefix to `story`, and we might want to re use the mask for other features : )